frontend/src/pages/profile.jsx

```diff
@@ -1,116 +1,175 @@
 import React from 'react';
 import './profile.css';
+import axios from 'axios'
+import { useState } from 'react'
+import { useEffect } from 'react';
 
 const ProfilePage = () => {
+ const userId = '6563a79964a01b13172d34ab'; //Josie Bruin
+//  const userId = '6563cef6161e4bbdde7d8c3c'; //Bob Brown
```

Comment:
Everything looks good and works! Maybe delete any comments that are not needed anymore!